code/script.js

```diff
@@ -0,0 +1,80 @@
+const avatar = document.getElementById('avatar')
+const projects = document.getElementById('projects')
+const userInfo = document.getElementById('userinfo')
+const username = 'Katarina821'
+
+const API_URL_REPOS = `https://api.github.com/users/${username}/repos`
+const API_URL_ID = `https://api.github.com/users/${username}`
+
+
+
+const API_TOKEN = TOKEN || process.env.API_KEY;
+
+const options = {
+    method: 'GET',
+    headers: {
+        Authorization: `token ${API_TOKEN}`
+    }
+}
+
+
+
+//First function, fetches userdata and displays them in userinfo
+const getIntro = () => {
+	fetch(API_URL_ID, options).then(res => res.json()).then(data => {
+		userInfo.innerHTML = `<h2>${data.name}</h2><a href=${data.html_url}> <h2> <img class="logo"src="./GitHubMark.png"> ${data.login} </h2></a>`
+		avatar.src = data.avatar_url
```

Comment:
U could write ` <img class="profile-img" src="${user.avatar_url}"> ` inside of the backticks, instead of declaring a variable above. Since you should be able to fetch that from the API: API_URL_ID. Or at least it did for me.